docs/manuals/source/IntroductionAndTutorial/WhatIsBareos.rst

```diff
@@ -158,6 +161,78 @@ Not all packages are required to run Bareos.
 
 -  On a Backup Administration system you need to install at least **bareos-bconsole** to have an interactive console to the |dir|.
 
+
+.. _section-BareosBinaryReleasePolicy:
+
+.. _section-BareosRepositoryTypes:
+
+Bareos Binary Release Policy
+----------------------------
+
+The Bareos project is fully committed to Open-Source.
+
+There are different types of Bareos binaries:
+
+#. Bareos Community binaries on https://download.bareos.org/:
+
+   * Publically accessable
+   * Latest build of the most recent Bareos stable branch at https://download.bareos.org/current/
+
+      * Only the latest build is available.
+      * Packages may be marked as prereleases (``<next-version-number>~pre``). However, packages are only published when they succeeded an automated testing process.
+      * When a new Bareos major version gets released, the version in this repository will also change to the new version.
+
+   * Latest build of the Bareos master branch at https://download.bareos.org/next/
+
+      * Only the latest build is available.
+      * Packages are marked as prereleases (``<next-major-version-number>.0.0~pre``). However, packages are only published when they succeeded an automated testing process.
+      * When a new Bareos major version gets released, the version in this repository will also change to the new version.
+
+#. Bareos Subscription binaries on https://download.bareos.com/
+
+   * Contain the repositories for the Bareos Subscription customers.
+   * The last three major releases are maintained and available in the corresponding subdirectory under https://download.bareos.com/bareos/release/
+   * Older versions stay available.
+   * While the repository can be browsed, using them do require authentication credentials, which come with a Bareos subscription.
```

Comment:
```suggestion
   * The last three major releases are maintained in https://download.bareos.com/bareos/release/
   * The repostories contain the whole binary history of each release, so that downgrades are possible if required.
   * Repositories of discontinued versions stay available.
   * While the repository can be browsed, accessing the binaries requires a Bareos Subscription.
```

Reply:
Maybe it makes sense to put the info where to buy the subscription here?